content/news/buzz/2025-curtin-capstone-year-end-blog/contents.lr

```diff
@@ -0,0 +1,135 @@
+
+To access the related resources for this deliverable:
+- View the [issue ticket](https://github.com/beeware/briefcase/issues/2337)
+- View the Toga pull request [here](https://github.com/beeware/toga/pull/3666)
+- View the Briefcase pull request [here](https://github.com/beeware/briefcase/pull/2442)
+- View the briefcase-web-static-template pull request [here](https://github.com/beeware/briefcase-web-static-template/pull/21)
+- View the documentation pull request [here](https://github.com/beeware/briefcase/pull/2511)
+
+## Other Contributions
+
+Below is a summary of contributions either made during the first semester or in down time during the second semester.
+
+### Briefcase Contributions
+
+- PR [#2198](https://github.com/beeware/briefcase/pull/2198): Add boolean question (feature)
+- PR [#2103](https://github.com/beeware/briefcase/pull/2203): Add XML content escaping filter to cookiecutter.py (feature)
+- PR [#2199](https://github.com/beeware/briefcase/pull/2199): Add catch exception when deleting JDK (bugfix)
+- PR [#2229](https://github.com/beeware/briefcase/pull/2229): Accept other changelog name and extension (feature)
+- PR [#2201](https://github.com/beeware/briefcase/pull/2201): Add ``--app`` option to briefcase build/package (feature)
+- PR [#2214](https://github.com/beeware/briefcase/pull/2214): Add ``--app`` option to briefcase create/update (feature)
+- PR [#2236](https://github.com/beeware/briefcase/pull/2236): Normalise contribution docs with Toga (documentation)
+- PR [#54](https://github.com/beeware/briefcase-windows-VisualStudio-template/pull/54): Use XML content escaping filter to Visual Studio template. (bugfix)
+
+### Toga Contributions
+
+- PR [#3259](https://github.com/beeware/toga/pull/3259): Add web screenshots (documentation)
+- PR [#3466](https://github.com/beeware/toga/pull/3466): Fix button click handling in Toga Web backend to correctly trigger event (bugfix)
+- PR [#3338](https://github.com/beeware/toga/pull/3338): DateInput (web widget)
+- PR [#3405](https://github.com/beeware/toga/pull/3405): TimeInput (web widget)
+- PR [#3362](https://github.com/beeware/toga/pull/3362): ScrollContainer (web widget)
+- PR [#3425](https://github.com/beeware/toga/pull/3425): Table (web widget) (work in progress)
+- PR [#3402](https://github.com/beeware/toga/pull/3402): Selection (web widget)
+- PR [#3527](https://github.com/beeware/toga/pull/3527): Slider (web widget)
+- PR [#3788](https://github.com/beeware/toga/pull/3788): MultilineTextInput (web widget)
+
+## Lessons Learnt
+
+This year with BeeWare has marked many valuable lessons taught outside of the class environment. Each of us have now made significant contributions to an open-source project while meeting deadlines and attending weekly standup meetings. This process has allowed us to develop our skills in numerous ways. Below each of us will comment our individual lessons.
+
+### Kavidu:
```

Comment:
You'll need to add a blank line between paragraphs to ensure this renders *as* paragraphs.